.github/taskfiles/quality.yml

```
@@ -0,0 +1,41 @@
+    desc: Check for dependency issues
+    cmds:
+      - task build:uv -s
+      - printf "${BLUE}[INFO] Running deptry...${RESET}\n"
+      - |
+        if [ -f "pyproject.toml" ]; then
+          SOURCE_FOLDER="src/$(find src -mindepth 1 -maxdepth 1 -type d -not -path '*/\.*' | head -1 | sed 's|^src/||')"
+          uvx deptry $SOURCE_FOLDER {{.OPTIONS}}
+        else
+          printf "${YELLOW}[WARN] No pyproject.toml found, skipping deptry${RESET}\n"
+        fi
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)